run_mypy.py

```diff
@@ -64,10 +32,10 @@ def main() -> int:
 
     print('Running mypy (this can take some time) ...')
     p = subprocess.run(
-        [sys.executable, '-m', 'mypy'] + args + modules,
+        [sys.executable, '-m', 'mypy'] + args,
         cwd=root,
     )
     return p.returncode
 
 if __name__ == '__main__':
-    sys.exit(main())
+    raise SystemExit(main())
```

Comment:
if we're just going to put the arguments in the .mypy.ini anyway, and not use run_mypy in CI, do we even need the run_mypy script? @mensinda, opinions?

Reply:
The run_mypy.py benefit (ability to run mypy from any directory) can be accomplished by

```sh
mypy --config-file /path/to/meson/.mypy.ini
```

Reply:
> The run_mypy.py benefit (ability to run mypy from any directory)

Why is this even considered "a benefit"?

Reply:
Well, at least for me, when I am working in the `work area` and I can just run `../run_mypy.py`